Makefile

```diff
@@ -10,13 +10,33 @@ datadir = ${datarootdir}
 docdir = ${datarootdir}/doc/${PACKAGE_TARNAME}
 sysconfdir = ${prefix}/etc
 mandir=${datarootdir}/man
+
+pkgdatadir = ${datadir}/${PACKAGE_TARNAME}
+cowsdir = ${pkgdatadir}/cows
+sitecowsdir = ${pkgdatadir}/site-cows
+
+pkgsysconfdir = ${sysconfdir}/${PACKAGE_TARNAME}
+cowpathdir = ${pkgsysconfdir}/cowpath.d
+
 srcdir = .
 
+A2X = a2x
 SHELL = /bin/sh
-INSTALL = install
+INSTALL = install -c
```

Comment:
Okay, one last thing: because `INSTALL` is defined as `install -c` and `INSTALL_DIRS` is defined recursively using `$(INSTALL)`, `INSTALL_DIRS` ends up being `install -c -d`.

```
[cowsay] $ make install
install -c -d /usr/local/etc/cowsay/cowpath.d
install -c -d /usr/local/bin
install -c cowsay /usr/local/bin/cowsay
rm -f /usr/local/bin/cowthink
ln -s cowsay /usr/local/bin/cowthink
install -c -d /usr/local/share/man/man1
```

There's probably no effect to that, but it doesn't seem quite right to me?

On the other hand, it seems to be what GNU itself is doing: here's what the Makefile from [GNU coreutils](https://www.gnu.org/software/coreutils/) 9.1 looks like after I've run `configure`.

```
INSTALL = $(cu_install_program) -c
INSTALL_DATA = ${INSTALL} -m 644
INSTALL_PROGRAM = ${INSTALL}
INSTALL_SCRIPT = ${INSTALL}
[...]
MKDIR_P = /opt/local/bin/gmkdir -p
[...]
mkdir_p = $(MKDIR_P)
```

So they're not even using `install` for creating directories, and combining the `-d` and `-c` options isn't an issue for them. And maybe it's just using `install`'s `-c` option because `./configure` detected that I'm running on macOS or BSD, so it's doing that for compatibility with an old-style, non-GNU `install` command? I dunno. What do you think we should do here?

Maybe we should just drop the `-c` option to `install` here and assume all modern installs have the copy-by-default behavior? I'm not about to suggest we slap on a GNU Autotools layer here; that's way overkill for cowsay.

Also the [GNU Makefile Conventions](https://www.gnu.org/prep/standards/html_node/Command-Variables.html#Command-Variables) say to have an INSTALL_PROGRAM variable and use that instead of using `$(INSTALL)` directly for installing executables. Let's do that.

Sorry for being so nitpicky here, but I think the spirit of this PR is "let's Do This 'The Right Way'", and think you're approaching it in a spirit of fun like I am. 😃